packages/attribute-registry-readmodel-writer/package.json

```diff
@@ -29,7 +29,6 @@
     "vitest": "1.6.0"
   },
   "dependencies": {
-    "@jm18457/kafkajs-msk-iam-authentication-mechanism": "3.1.2",
```

Comment:
Somehow unrelated - the attribute-registry-readmodel-writer was using its own custom authentication mechanism for Kafka instead of relying on the one from commons 🤔🤔🤔  
Let's use this PR as a chance to align it to other reamodel-writer services

Reply:
@galales / @beetlecrunch if we want to bring this specific change into this release, I could create a separate PR for it

Reply:
yes please 🙏 , we need to deploy the use of the aws library in this release for all consumers, but probably we will shift the merge of the config refactor next week

Reply:
Change moved in #671